architecture/add/05-architecture-security.md

```diff
@@ -100,8 +100,7 @@ flow.
 Auth0 provided database but intend to extend this to include other types of connections
 if needed by our target organisations.
 * **Extensions** - Auth0 provides an extension mechanism and a library of existing
-extensions. We are using the [Auth0 Authorisation](https://auth0.com/docs/extensions/authorization-extension/v2)
-extension to provide an easy way to define roles and assign them to users.
+extensions. 
```

Comment:
The later extensions section has been removed so I assume we are no longer using any Auth0 extensions? I think this list is meant to be a list of the Auth0 functionality being used, so should it be removed from here too?